openstack/blockstorage/v3/volumetypes/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package volumetypes provides information and interaction with volume types in the
+// OpenStack Block Storage service. A volume type is a collection of specs used to
+// define the volume capabilities.
```

Comment:
Can you structure this similar to the other `doc.go` files and include an example of how to use this package? Each PR will amend the example.